### DISCUSSION TITLE
指定debug mode下载二进制文件的debug info里没有源码路径信息

### DISCUSSION THREAD

**Discussion by Brubbish** [xmake-io/xmake-repo#5857] [Q&A]
我使用`xrepo install -k shared -m debug libpng、zlib、7z、pcre2`下载了四个项目的文件进行测试，在IDA Pro里打开都没有源码路径信息，但确实是not stripped和with debug info的。
![image](https://github.com/user-attachments/assets/7ff75057-4596-4a51-891c-7fecd419634d)
![image](https://github.com/user-attachments/assets/7d7ec79e-b85f-4ff3-b495-dfac93b815c6)
我期望的效果：
![image](https://github.com/user-attachments/assets/2e6a5772-0f30-4924-971c-84c29e2e8172)
想知道如何获取预编译的带源码路径的文件；或者把源码路径strip的原因是什么

**Comment by saitou-yuki** [comment]
- 前営業日にやったこと
  - 定常タスク
    - PRレビュー
  - ネイティブアプリ
    - SecretsManagerへの登録依頼(金子さん)
    - https://github.com/daytona-international/di-ddp/issues/27014
      - https://github.com/daytona-international/di-ddp/issues/27018
    - https://github.com/daytona-international/di-ddp/issues/27215
      - https://github.com/daytona-international/di-ddp/issues/27217
- 今日やること
  - 定常タスク
    - [ ] PRレビュー
  - ネイティブアプリ
    - https://github.com/daytona-international/di-ddp/issues/27014
      - [ ] https://github.com/daytona-international/di-ddp/issues/27018
    - https://github.com/daytona-international/di-ddp/issues/27215
      - [ ] https://github.com/daytona-international/di-ddp/issues/27217
      - [ ] https://github.com/daytona-international/di-ddp/issues/27317
    - 残タスク
      - [ ] 商品検索API - キャンペーン・クーポン・商品価格周りの実装
      - [ ] 商品詳細API - キャンペーン・クーポン・商品価格周りの実装
      - [ ] https://github.com/daytona-international/di-ddp/issues/27220
    - 保留中
      - [ ] ZETA_SEARCH:価格帯検索時の参照項目について確認
        - [To: ZETA様](https://daytonajp.backlog.com/view/APP_RENEWAL-2842)
- 困っていること
- 滞留PR報告
  - di-docs
    - https://github.com/daytona-international/di-docs/pull/1989
